source/process/gitlab-process.rst

```diff
@@ -0,0 +1,116 @@
+
+ - Pre-provisioning OAuth configuration automatically on the Omnibus package
+ - Mattermost ChatOps slash command integration
+ - OAuth team creation option in GitLab Omnibus
+
+Communication
+-------------------
+
+Changes and features affecting GitLab Omnibus are communicated via `GitLab issues <https://gitlab.com/gitlab-org/gitlab-ce/issues>`_ and `GitLab Slack channel <https://gitlab.slack.com>`_.
+
+Moreover, a recurring monthly meeting between a product manager at GitLab and Mattermost is organized to cover
+
+ - concerns/issues from GitLab Omnibus team,
+ - new upcoming changes or features in Mattermost affecting GitLab Omnibus, and
+ - other miscellaneous queued items.
+
+Optionally, an engineer from both GitLab and Mattermost teams also joins the recurring monthly meeting.
+
+Service-Level Agreement (SLA)
+-------------------------------
+
+This process is still a work-in-progress, being developed together with GitLab.
+
+GitLab Premier Support Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GitLab has a `4-hour support window for their premier support customers <https://about.gitlab.com/features/premium-support/>`_. Mattermost is setting up a process to meet those requirements for relevant questions raised by the GitLab premier support customers.
+
+GitLab is also working with their legal team to understand the process for sharing any sensitive customer information (e.g. logs) with Mattermost.
+
+More details to be added.
+
+GitLab Non-Premier Support Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost has created a `mattermost-support` account in GitLab for support issues. The plan is for Mattermost to
+
+- integrate GitLab issue reviews as part of existing forum issue monitoring processes,
+- aim to answer within one business day, or at best effort, and
+- subscribe to email alerts sent automatically when a `Mattermost` label is added to an issue, or when the `mattermost-support` account is assigned to a GitLab issue.
+
+Templates
+--------------
+
+GitLab announcement proposal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+  Proposed update for new version of [Mattermost](https://gitlab.com/gitlab-org/omnibus-gitlab/issues/1241).
+
+  ## GitLab Mattermost 2.2
+
+  [Mattermost 2.2](http://www.mattermost.org/mattermost-2-2-threaded-messages-and-more/) ships in GitLab 8.7 with threaded messages, French translation, new themes, new Trello and IRC support, plus many more new benefits.
+
+  This version also includes [security updates](http://about.mattermost.com/security-updates/) and upgrade from earlier versions is recommended.
+
+GitLab forum responses
+~~~~~~~~~~~~~~~~~~~~~~~
+
+See `sample forum responses listed here <https://docs.mattermost.com/process/community-guidelines.html#sample-responses>`_.
+
+Upcoming Work
```

Comment:
We're going to have to make sure to keep this list up to date since it may easily get out of sync with the actual work that needs to be done

Reply:
@hmhealey Yeah, I'll try and keep the section updated. We probably won't need it once GitLab ships with ESR in a few months, since most of the work would be done by then.